packages/babel-plugin-import-jsx-pragma/package.json

```
@@ -6,7 +6,9 @@
 	"license": "GPL-2.0-or-later",
 	"keywords": [
 		"wordpress",
-		"babel-plugin",
+		"gutenberg",
+		"babel",
+		"plugin",
```

```suggestion
		"plugin",
		"babel-plugin",
```

babel-plugin also seems a [common label](https://www.npmjs.com/search?q=keywords:babel-plugin)